src/extensions/replay/sessionrecording.ts

```diff
@@ -619,7 +619,7 @@ export class SessionRecording {
             clearInterval(this._fullSnapshotTimer)
         }
 
-        this._fullSnapshotTimer = setInterval(() => {
+        this._fullSnapshotTimer = window?.setInterval(() => {
```

Comment:
feels like a safe enough change so 🚢 

but definitely be good to understand why this works in CI, and on my machine, but not on yours

Reply:
That's surprising... I will clear my local node_modules or something and try again...

Reply:
ah ha. The error went away!